packages/babel-types/src/definitions/index.ts

```diff
@@ -28,6 +29,19 @@ import { DEPRECATED_ALIASES } from "./deprecated-aliases.ts";
     FLIPPED_ALIAS_KEYS[DEPRECATED_ALIASES[deprecatedAlias]];
 });
 
+for (const { types, set } of allExpandedTypes) {
+  for (const type of types) {
+    const aliases = FLIPPED_ALIAS_KEYS[type];
+    if (aliases) {
+      for (const alias of aliases) {
+        set.add(alias);
+      }
```

Comment:
```suggestion
      aliases.forEach(set.add, set);
```

maybe this is faster, since it doesn't go through the iterator protocol?

Reply:
https://leanylabs.com/blog/js-forEach-map-reduce-vs-for-for_of/
The callback seems to be slower. :)

Reply:
Well that blog post is with an inline closure -- they say:

>  It’s worth using only when you already have a function to invoke on every array element. In this case, it’s a one-liner, with zero performance degradation:
>
> ```js
> array.forEach(func);
> ```